### DISCUSSION TITLE
How to disable Docs menu entry in Storybook?

### DISCUSSION THREAD

**Discussion by ackvf** [storybookjs/storybook#22810] [Help]
Prior to version 7, the "Docs" were available as a tab at the top *after* opening a story from sidebar like this:  
[![enter image description here][2]][2]

However in version 7, "Docs" were removed from the tab and are now available from the sidebar menu, which clutters the menu space. How can I override and disable this per story or globally? Can I switch back to old way (Docs as a card)?

before / after  
[![enter image description here][1]][1]

  [1]: https://i.stack.imgur.com/KnBHG.png
  [2]: https://i.stack.imgur.com/fjXbI.png

**Comment by f3bw** [comment]
Hi there,

Not sure if you can get the Docs tab at the position it was prior SB7, still looking into that.


In Storybook 7, you can turn off the Docs addon by modifying the configuration. Here's how you can do it:

1. Locate the `.storybook/main.js` file in the root directory of your project.

2. Open the file and find the `addons` section. It should look something like this:

   ```javascript
   module.exports = {
     addons: [
       // Other addons...
       '@storybook/addon-docs',
     ],
   };
   ```

3. To disable the Docs addon, you can simply comment out or remove the `'@storybook/addon-docs'` entry from the addons array:

   ```javascript
   module.exports = {
     addons: [
       // Other addons...
       // '@storybook/addon-docs',
     ],
   };
   ```

4. Save the changes to the file.

5. Restart your Storybook server, and the Docs addon should no longer be active.

**Reply by ackvf** [reply]
I don't really want to disable it completely, but merely remove it from being the first default option when opening a story and removing it from the sidebar where it clutters the space a lot. It makes it into 3x the amount of items in there 🤯🤯

**Reply by k-sav** [reply]
I want to do this too. Because it tries to render too many stories at once and crashes the browser

**Comment by cmawhorter** [comment]
I landed here through search and this seems like the best place to put this --

I think storybook/addon-docs was moved into sb core? Not sure, but I no longer have a separate addon.

This is what I did to get the "Docs" entry for each story removed:
- add `docs: {autodocs: false}` in my ".storybook/main.ts" even though it says it's deprecated
- add `tags: ['!autodocs']` to ".storybook/preview.ts"
- make sure you don't have an `@storybook/addon-docs` entry in main.ts addons
- restart storybook

I tried just editing tags in preview like people were suggesting and it didn't work for me. However, updating both spots and restarting storybook did, and now autodocs are gone.

**Reply by shilman** [reply]
I would expect that if all your `@storybook/*` packages are up to date, all you need to do is add `tags: ['!autodocs']` to `.storybook/preview.ts`. Any chance you can verify that for me?
